python/test/unit/cuda/test_gemm.py

```diff
@@ -455,6 +455,6 @@ def grid(META):
         ptx = pgm.asm['ptx']
         if is_tcgen5:
             assert re.search(r'tcgen05.mma.cta_group::1.kind::f16', ptx)
-        else:
+        elif torch.cuda.get_device_capability()[0] == 10:
             wgmma_n = int(max(BLOCK_N / max(NUM_WARPS / max(BLOCK_M / 16, 1), 1), 8))
             assert re.search(r'wgmma.mma_async.sync.aligned.m\d+n{}k16(?:.row.col)?.f32.f16.f16'.format(wgmma_n), ptx)
```

Comment:
What's going on here? Blackwell doesn't have wgmma.

Reply:
Should be `== 9`?

Reply:
Oops! Typo, that should have been 9, will fix.

Reply:
fixed, thanks!